registry/registry.go

```diff
@@ -319,7 +322,10 @@ func (r *RouteRegistry) pruneStaleDroplets() {
 
 	r.byURI.EachNodeWithPool(func(t *container.Trie) {
 		endpoints := t.Pool.PruneEndpoints()
-		t.Snip()
+		if time.Now().Sub(t.Pool.LastUpdated()) > t.Pool.EmptyPoolTimeout {
```

Comment:
```suggestion
		if time.Since(t.Pool.LastUpdated()) > t.Pool.EmptyPoolTimeout {
```
time.Since() is recommended instead of time.Now().Sub(other) for legibility.